fileio/ft_fopen.m

```diff
@@ -0,0 +1,20 @@
+function fid = ft_fopen(filename, mode, varargin)
```

Comment:
Please call it fopen_or_error and move it to the private directories.

Reply:
Done.